src/App.js

```diff
@@ -17,6 +17,9 @@ export default function App() {
           <Route exact path="/apply" render={() => {
             window.location.href="/pages/apply.html"
             }} />
+          <Route exact path="/pages/contact" render={() => {
+            window.location.href="/pages/contact.html"
+            }} />
```

Comment:
There were a few pages that routed to 404 for me locally, but work in prod. Anyone know why? (I'm happy to remove this once I figure out why.)

Reply:
No idea. Will this route to https://refcode.org/pages/contact once pushed live?